### PR TITLE
Migrate critical gce and kubemark jobs

### PR DESCRIPTION
**Pull request by fejta**
ref #1021 #1058 

Similar contrib PR: https://github.com/kubernetes/contrib/pull/2047

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1089)

### CODE REVIEW

**Comment by krzyzacy** [comment]
oh lol I did not realize I've already committed the gce jobs

File: jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml [hunk 1]
```diff
@@ -52,22 +52,22 @@
...
     # START GCE
-#    - kubernetes-e2e-gce:
-#        job-name: ci-kubernetes-e2e-gce
-#        frequency: 'H/5 * * * *' # At least every 30m
-#        trigger-job: 'kubernetes-build'
-#    - kubernetes-e2e-gci-gce:
-#        job-name: ci-kubernetes-e2e-gci-gce
-#        frequency: 'H/5 * * * *' # At least every 30m
-#        trigger-job: 'kubernetes-build'
+     - kubernetes-e2e-gce:
```
Comment (krzyzacy) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml]: move it back one space :-)

Comment (fejta) [inline jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml]: Done

**Comment by krzyzacy** [comment]
Non-automatic merge from submit-queue

**Pull request merged**